### PR TITLE
add bugsnag source maps dependency

### PR DESCRIPTION
**Pull request by dandlezzz**
CI flow for bugsnag

### CODE REVIEW

File: .github/workflows/sourcemaps.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  upload_source_map:
+    runs-on: ubuntu-latest
+    name: Upload Source Maps to Bugsnag
+    run: npx bugsnag-source-maps upload-node --api-key bd22af1f4e1213b5badb590a48128bf1 --directory=./.next/static/chunks/
```
Comment (kespinola) [inline .github/workflows/sourcemaps.yml:12]: I believe this is a more sensitive api key than the one given to the web client. Can you move it to a GitHub secret and rotate?

**Review by kespinola — COMMENTED** [review]
Thinking more on this.  Heroku is building the client app. I'm wondering if we can plug into that to do the upload of source maps to bugsnag

File: .github/workflows/sourcemaps.yml (hunk 1 above)
Comment (dandlezzz) [reply .github/workflows/sourcemaps.yml:12]: funnily enough its the same one.

**Comment by dandlezzz** [comment]
@kespinola yeah, this is a better thing for heroku. I think.

**Review by kespinola — APPROVED** [review]

**Pull request merged**